templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/cocos/sys-ability-polyfill.js

```diff
@@ -167,5 +171,46 @@ globalThis.vibrate = function (duration) {
 }
 
 globalThis.terminateProcess = function () {
-    this.pro.exit(0);
+    pro.exit(0);
```

Comment:
When do we need to exit a process?

Reply:
When to quit the game, cc.game.end()  will call it.